api/scenario_api_utils/launch/clock_publisher.launch.xml

```diff
@@ -3,7 +3,7 @@
 
   <arg name="time_rate" default="0.5" />
 
-  <node pkg="scenario_api_utils" type="ros_time_publisher.py" name="ros_time_publisher" output="log">
-    <param name="~time_rate" value="$(arg time_rate)" />
+  <node pkg="scenario_api_utils" exec="ros_time_publisher.py" name="ros_time_publisher" output="log">
+    <param name="~time_rate" value="$(var time_rate)" />
```

Comment:
I think you would have to remove `~` and I tried but even then, the default time rate parameter listed in this file is not respected when I do

    ros2 launch scenario_api_utils clock_publisher.launch.xml

And changing it manually doesn't have any effect, either because there is no callback. But perhaps you don't care about that

    ros2 param set /ros_time_publisher time_rate 3.3

Reply:
Thanks for pointing out. I removed `~` in the launch file.
It shouldn't have to change time_rate online so I think it's okay.